src/RGUnityBots/Assets/Scripts/ActionManagerTests/InputSysKeyListeningObject.cs

```diff
@@ -0,0 +1,43 @@
+﻿
+
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace ActionManagerTests
+{
+    public class InputSysKeyListeningObject : MonoBehaviour
+    {
+        private Key fireKey = Key.Backspace;
+    
+        void Update()
+        {
+            if (Keyboard.current[Key.F2].isPressed)
+            {
+                Debug.Log("Keyboard.current[Key.A].isPressed");
```

Comment:
We should use `RGDebug` instead of `Debug` in all our SDK code

Reply:
Normally I use RGDebug, but this particular line of code is for a test scene in the RGUnityBots project. Eventually I want to use LogAssert.Expect to write automated tests that the action manager is covering all these input types, so I want to avoid the extra data that is introduced by RGDebug.